common/src/main/java/net/william278/huskclaims/command/ClaimPrivateCommand.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of HuskClaims, licensed under the Apache License 2.0.
+ *
+ *  Copyright (c) William278 <will27528@gmail.com>
+ *  Copyright (c) contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package net.william278.huskclaims.command;
+
+import net.william278.huskclaims.HuskClaims;
+import net.william278.huskclaims.claim.Claim;
+import net.william278.huskclaims.claim.ClaimWorld;
+import net.william278.huskclaims.trust.TrustLevel;
+import net.william278.huskclaims.user.OnlineUser;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+
+public class ClaimPrivateCommand extends InClaimCommand {
+
+    protected ClaimPrivateCommand(@NotNull HuskClaims plugin) {
+        super(List.of("claimprivate"), "", TrustLevel.Privilege.MANAGE_BANS, plugin);
```

Comment:
Just thinking about this privilege here - I think we should add a separate privilege called `MAKE_PRIVATE` for a bit more customizability there (default to being usable by only the claim owner).

Reply:
Added the privilege